docs/source/overview/delivery_services.rst

```diff
@@ -0,0 +1,755 @@
+	+============================+=========================================================+=========================================+
+	| longDesc2\ [#cardinality]_ | Traffic Control source code and :ref:`to-api` responses | unchanged (``string``, ``String`` etc.) |
+	+----------------------------+---------------------------------------------------------+-----------------------------------------+
+
+Match List
+----------
+A Match List is a set of regular expressions used by Traffic Router to determine whether a given request from a client should be served by this Delivery Service. Under normal circumstances this field should only ever be read-only as its contents should be generated by Traffic Ops based on the Delivery Service's configuration. These regular expressions can each be one of the following types:
+
+HEADER_REGEXP
+	This Delivery Service will be used if an HTTP Header/Value pair can be found in the clients request matching this regular expression.\ [#httpOnlyRegex]_
+HOST_REGEXP
+	This Delivery Service will be used if the requested host matches this regular expression. The host can be found using the ``Host`` HTTP Header, or as the requested name in a DNS request, depending on the `Type`_ of the Delivery Service.
+PATH_REGEXP
+	This Delivery Service will be used if the request path matches this regular expression.\ [#httpOnlyRegex]_
+STEERING_REGEXP
+	This Delivery Service will be used if this regular expression matches the xml_id_ of one of this Delivery Service's "targets"
+
+		.. note:: This regular expression type can only exist in the Match List of STEERING-`Type`_ Delivery Services - and **not** CLIENT_STEERING.
+
+.. table:: Aliases
+
+	+-----------------------+----------------------+------------------------------------------------------+
+	| Name                  | Use(s)               | Type(s)                                              |
+	+=======================+======================+======================================================+
+	| deliveryservice_regex | Traffic Ops database | unique, integral identifier for a regular expression |
+	+-----------------------+----------------------+------------------------------------------------------+
+
+Max DNS Answers
+---------------
+The maximum number of :term:`Edge-tier cache server` IP addresses that the Traffic Router will include in responses to DNS requests for DNS-:ref:`Routed <ds-types>` Delivery Services. The :ref:`to-api` restricts this value to the range [1, 15], but no matching restraints are placed on the actual data as stored in the Traffic Ops Database. When provided, the :term:`cache server` IP addresses included are rotated in each response to spread traffic evenly. Ideally this number will reflect the amount of traffic - e.g. ``1`` for a trial Delivery Service with very little traffic, ``2`` for a small production Delivery Service. Add 1 for every 20 :abbr:`Gbps (Gigabits per second)` of traffic you expect at peak.
```

Comment:
The 20Gbps number is highly dependent upon the cache's networking capabilities. I don't think we should be making recommendations like this in the documentation. I think it's sufficient enough to say that this number should be scaled with the amount of traffic you expect from this delivery service.

Reply:
I agree - that's just a copy of what was under "Using".